inputs/select.vue

```diff
@@ -132,22 +132,36 @@
           value: 'value'
         }, this.args.keys || {});
       },
+      equalLabelValueKeys() {
+        return this.keys.label === this.keys.value;
+      },
+      valueKey() {
+        return this.equalLabelValueKeys ? 'value' : this.keys.value;
+      },
       NULL_OPTION() {
         return {
           [this.keys.label]: 'None',
-          [this.keys.value]: null
+          [this.valueKey]: null
         };
       },
       // combine arg/prop options, fetched list options, and a null option for non-multiple selects
       options() {
         const propOptions = this.args.options || [],
           currentSlug = _.get(this.$store, 'state.site.slug'),
+          { label } = this.keys,
           noneOption = propOptions.find((val) => {
-            return val.name === 'None' || val.name === 'Default';
+            return val[label] === 'None' || val[label] === 'Default';
           });
 
         let fullOptions = propOptions.concat(this.listOptions);
 
+        if (this.equalLabelValueKeys) {
+          fullOptions = fullOptions.map((option) => ({
+            [label]: option[label],
+            value: option[label]
```

Comment:
We can avoid having the if condition in here and just use `this.valueKey`
```suggestion
            [this.valueKey]: option[label]
```